test/common/cdp.py

```diff
@@ -44,7 +44,6 @@ def _path(self, show_browser):
         Reimplement this in subclasses, so it is easier to return None
         than to raise the proper exception (done at once in path()).
         """
-        pass
```

Comment:
That on the other hand is okay. It could raise a `NotImplementedError`, unless `@abstractmethod` already does?